core-services/prow/02_config/openshift/oadp-operator/_prowconfig.yaml

```diff
@@ -5,4 +5,5 @@ branch-protection:
         oadp-operator:
           branches:
             ci-multi-cloud:
-              protect: false
+              unmanaged: true
+          protect: true
```

Comment:
No need to have `protect: true` at all, it's gonna be driven by Prow jobs

Reply:
I added because of 
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/26420/pull-ci-openshift-release-master-config/1495766145582501888#1:build-log.txt%3A10

Reply:
Cool, TIL